src/middle-earth/str_and_repr.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+
+class MyCoolClass:
+    def __init__(self, *, value: str) -> None:
```

Comment:
```suggestion
    def __init__(self, value: str) -> None:
```
Is making the argument keyword-only really necessary?